api/src/test/java/edu/cornell/mannlib/vitro/webapp/rdfservice/RDFServiceNotifictaionTest.java

```diff
@@ -0,0 +1,132 @@
+package edu.cornell.mannlib.vitro.webapp.rdfservice;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.jena.ontology.OntModel;
+import org.apache.jena.ontology.OntModelSpec;
+import org.apache.jena.rdf.listeners.StatementListener;
+import org.apache.jena.rdf.model.ModelChangedListener;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.junit.Before;
+import org.junit.Test;
+
+import edu.cornell.mannlib.vitro.webapp.dao.jena.event.BulkUpdateEvent;
+import edu.cornell.mannlib.vitro.webapp.rdfservice.impl.RDFServiceUtils;
+import edu.cornell.mannlib.vitro.webapp.rdfservice.impl.jena.model.RDFServiceModel;
+
+public class RDFServiceNotifictaionTest {
```

Comment:
there is a typo in the name of the class - Notification

Reply:
fixed